backend/src/seeds/users/bibi-bloxberg.ts

```diff
@@ -8,4 +8,5 @@ export const bibiBloxberg: UserInterface = {
   emailChecked: true,
   language: 'de',
   publisherId: 1234,
+  password: 'Aa12345_',
```

Comment:
Why? Is there a case we want another password in the seeded data?

Reply:
the password is used in the contribution factory to login the user